helm/bailo/templates/bailo/bailo.configmap.yaml

```diff
@@ -53,6 +67,9 @@ data:
           supportHref: 'mailto:{{ .Values.config.supportEmail }}?subject=Bailo%20Support',
           contactHref: 'mailto:{{ .Values.config.supportEmail }}?subject=Bailo%20Contact',
         },
+        help: {
+          documentationUrl: '{{ .Values.config.documentationLink }}',
```

Comment:
This value doesn't appear to be used in the application anymore.

Reply:
removed